### PR TITLE
Update domain password policy in beta

### PR DESCRIPTION
**Pull request by eketo-msft**

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[eae7b13](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/eae7b13457431d0c29a4512f26446060cd1d5c72)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 24 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[eae7b13](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/eae7b13457431d0c29a4512f26446060cd1d5c72)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 2 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[eae7b13](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/eae7b13457431d0c29a4512f26446060cd1d5c72)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/domain.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-domain-password-policy/api-reference/beta/resources/domain.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-domain-password-policy/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C20%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902200238317683-4012%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=tlfUlr1kG8UE8fsQOqiy5Ww3gaMKpgnAhiKw8%2FWbUOA%3D&st=2019-02-20T02%3A37%3A16Z&se=2019-03-23T02%3A42%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[df07d64](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/df07d64c4a24eee9fbb7dea2ce77105a9d4079d6)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 24 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[df07d64](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/df07d64c4a24eee9fbb7dea2ce77105a9d4079d6)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 2 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[df07d64](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/df07d64c4a24eee9fbb7dea2ce77105a9d4079d6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/domain.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-domain-password-policy/api-reference/beta/resources/domain.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-domain-password-policy/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C20%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902200252551341-4012%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gszNv9tGvqq%2Bp34jRTTL5tx3Hnrg%2FwzJzfJdFmM%2BIzA%3D&st=2019-02-20T02%3A51%3A45Z&se=2019-03-23T02%3A56%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/resources/domain.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ To associate a domain with a tenant:
 |isInitial|Boolean| True if this is the initial domain created by Microsoft Online Services (companyname.onmicrosoft.com). There is only one initial domain per company. Not nullable |
 |isRoot|Boolean| True if the domain is a verified root domain. Otherwise, false if the domain is a subdomain or unverified. Not nullable |
 |isVerified|Boolean| True if the domain has completed domain ownership verification. Not nullable |
+|passwordNotificationWindowInDays|Int32|Specifies the number of days before a user receives notification that their password will expire. If the property is not set a default value of 14 days will be used.|
+|passwordValidityPeriodInDays|Int32| Specifies the length of time that a password is valid before it must be changed. If the property is not set a default value of 90 days will be used. |
```
Comment (dkershaw10) [inline api-reference/beta/resources/domain.md]: What do I set this to, if I don't want passwords to expire?  What's the max?

File: api-reference/beta/resources/domain.md [hunk 2]
```diff
@@ -52,6 +52,8 @@ To associate a domain with a tenant:
 |isInitial|Boolean| True if this is the initial domain created by Microsoft Online Services (companyname.onmicrosoft.com). There is only one initial domain per company. Not nullable |
 |isRoot|Boolean| True if the domain is a verified root domain. Otherwise, false if the domain is a subdomain or unverified. Not nullable |
 |isVerified|Boolean| True if the domain has completed domain ownership verification. Not nullable |
+|passwordNotificationWindowInDays|Int32|Specifies the number of days before a user receives notification that their password will expire. If the property is not set a default value of 14 days will be used.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/domain.md]: What's the max I can set this to?

Comment (dkershaw10) [inline api-reference/beta/resources/domain.md]: I know these properties are on domain, but I wonder whether we should also update the user passwordPolicies property to link to this domain level policy?

File: api-reference/beta/resources/domain.md (hunk 1 above)
Comment (eketo-msft) [reply api-reference/beta/resources/domain.md]: Office sets max int value here to indicate not expired.

File: api-reference/beta/resources/domain.md (hunk 2 above)
Comment (eketo-msft) [reply api-reference/beta/resources/domain.md]: Validation is not enforced.  You can set any valid Int32 value.

Comment (eketo-msft) [reply api-reference/beta/resources/domain.md]: I don't think it is needed.  PasswordPolicy and PasswordProfile don't specifically talk about notifcation or validity at all.  The coupling is loose at best, I think docs cross linking should strive to be strongly coupled between topics.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[5865eb4](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/5865eb4d1c402aacb1f52f58d013be5f9adc5871)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[5865eb4](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/5865eb4d1c402aacb1f52f58d013be5f9adc5871)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[5865eb4](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/5865eb4d1c402aacb1f52f58d013be5f9adc5871)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/domain.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-domain-password-policy/api-reference/beta/resources/domain.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-domain-password-policy/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C27%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902272320574975-4012%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=mE87nAypNlb0L8pO2BK1cfccaovs%2BPZZ4%2BKVYJSi2Qk%3D&st=2019-02-27T23%3A19%3A45Z&se=2019-03-30T23%3A24%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d70c6ce](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/d70c6cec2463c3e3607a4ad02c8ea2a517318171)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d70c6ce](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/d70c6cec2463c3e3607a4ad02c8ea2a517318171)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d70c6ce](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/d70c6cec2463c3e3607a4ad02c8ea2a517318171)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/domain.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-domain-password-policy/api-reference/beta/resources/domain.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-domain-password-policy/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C27%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902272322173335-4012%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=oMquC%2BnP74r1QHFcAPsWIjm1armMhR6fdUBrSJniHp4%3D&st=2019-02-27T23%3A23%3A24Z&se=2019-03-30T23%3A28%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/resources/domain.md (hunk 2 above)
Comment (dkershaw10) [reply api-reference/beta/resources/domain.md]: OK we can fix later.

File: api-reference/beta/resources/domain.md (hunk 1 above)
Comment (dkershaw10) [reply api-reference/beta/resources/domain.md]: OK we can fix later.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d9401fe](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/d9401fef7028230e428a6cb52cbd254048a9b4fe)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d9401fe](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/d9401fef7028230e428a6cb52cbd254048a9b4fe)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d9401fe](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/d9401fef7028230e428a6cb52cbd254048a9b4fe)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/domain.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-domain-password-policy/api-reference/beta/resources/domain.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-domain-password-policy/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C28%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902282239571888-4012%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=n5nPPCvhgpEu2h9Mm9EBAdY2W4OxfRTYFugJeBlpO64%3D&st=2019-02-28T22%3A38%3A26Z&se=2019-03-31T22%3A43%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[13e6698](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/13e6698388e7ed5f28e6bb21c0703c7bc7587cb3)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[13e6698](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/13e6698388e7ed5f28e6bb21c0703c7bc7587cb3)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[13e6698](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4012/commits/13e6698388e7ed5f28e6bb21c0703c7bc7587cb3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/domain.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-domain-password-policy/api-reference/beta/resources/domain.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-domain-password-policy/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C28%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902282240077067-4012%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=x6LybI9R%2FU9dsbJNTDFtkRb5PPGwC8P5eNesEvZZqRk%3D&st=2019-02-28T22%3A42%3A02Z&se=2019-03-31T22%3A47%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**